codes/quantum/spins/spins_into_spins.yml

```diff
@@ -8,12 +8,33 @@ physical: spins
 logical: spins
 
 name: 'Spin code'
```

Comment:
This is an umbrella entry for the Gross codes but also others. Please create a new entry for the J. Gross codes on next round.

Reply:
Does J. Gross codes refer to the example in "Encoding a qubit in a spin"

Reply:
Yes

Reply:
Please resubmit with the text you added here moved to a new entry for "Gross spin code" and you'll be all set. Thanks.